router/core/access_controller.go

```diff
@@ -44,3 +87,23 @@ func (a *AccessController) Access(w http.ResponseWriter, r *http.Request) (*http
 	}
 	return r, nil
 }
+
+func (a *AccessController) IntrospectionTokenAccess(r *http.Request, body []byte) bool {
+	if a.introspectionAuthMode == IntrospectionAuthModeToken {
+		authHeader := r.Header.Get("Authorization")
+		if authHeader == "" {
+			return false
+		}
+
+		authHeader = strings.TrimSpace(authHeader)
+
+		// guard to prevent bypass when token is unset
+		if a.introspectionAuthSkipToken == "" {
+			return false
+		}
```

Comment:
Can we fail this at setup instead (or as well?)

Reply:
There´s a schema check which ensures no empty tokens are provided when auth mode == token